components/index/SiteTopUpper/Consultation.vue

```diff
@@ -3,11 +3,13 @@
     <v-lazy>
       <static-info
         class="Consultation"
-        url="https://www.fukushihoken.metro.tokyo.lg.jp/iryo/kansen/coronasodan.html"
+        url="https://www.fukushihoken.metro.tokyo.lg.jp/iryo/kansen/corona_portal/index.html"
         :text="
-          $t('自分や家族の症状に不安や心配があれば\nまずは電話相談をどうぞ')
+          $t(
+            'ワクチン情報・変異株情報・検査情報等の\n新型コロナ関連情報はこちら'
```

Comment:
```suggestion
            'ワクチン情報・変異株情報・検査情報等の新型コロナ関連情報はこちら'
```

`\n` は翻訳時の事故のもとなので 💦  入れなくて大丈夫です！
`\n` をとった上で、cssのほうを整えてもらえると助かります 🙏

Reply:
@kaizumaki 
コメントありがとうございます！修正したものをpushしたのでご確認よろしくお願いします！

Reply:
@knknk98 ご対応ありがとうございます！
プレビュー見て思ったんですが、`flex-wrap: wrap;` を削除するだけだと、以下のようにブラウザ幅を狭くした際にちょっと見栄えが良くないのですよね...。

![スクリーンショット 2021-08-20 5 46 04](https://user-images.githubusercontent.com/14883063/130141821-37dc7961-2dc0-45bf-8f0f-677c97c09563.png)

.StaticInfoの `flex-wrap: wrap;` は削除せずに、
.StaticInfo-Textに以下を追記すると、程よいところで折り返してくれて、見栄えもよさそうです。
```
flex: 1 0 50%;
padding-right: 16px;
```
（padding-rightは、ボタンとの隙間を開けるために入れています）

ちょっと試してみてもらえると嬉しいです 🙏

Reply:
@kaizumaki 
ありがとうございます！確かにブラウザ幅狭くした際に見栄えが良くなかったですね...。
.StaticInfoの`flex-wrap: wrap;`を削除せず.StaticInfo-Textに

```
flex: 1 0 50%;
padding-right: 16px;
```
を追加したのでご確認よろしくお願いします！

Reply:
プレビュー見ました！いいですね！ありがとうございます！